packages/core/src/Layout/Layout.ts

```diff
@@ -41,21 +54,49 @@ class Layout {
       };
       pageSources = new PageSources();
       this.layoutPageBodyVariable = `{{${uuidv4()}-${uuidv4()}}}`;
-      const layoutVariables = { [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable };
+      const layoutVariables = {
+        [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable,
+      };
 
-      const { variableProcessor, pluginManager, siteLinkManager } = this.config;
+      const {
+        variableProcessor,
+        pluginManager,
+        siteLinkManager,
+      }: {
+        variableProcessor: VariableProcessor;
+        pluginManager: any;
```

Comment:
We can use the defined `PluginManager` type here - It's exported from `PluginManager.ts`